### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by jkuri**

### CODE REVIEW

File: server/scheduler/scheduler.go [hunk 1]
```diff
@@ -14,9 +14,9 @@ import (
 	"github.com/bleenco/abstruse/server/core"
 	"github.com/bleenco/abstruse/server/ws"
 	"github.com/drone/go-scm/scm"
+	"github.com/golang/protobuf/proto"
 	"github.com/logrusorgru/aurora"
 	"go.uber.org/zap"
-	"google.golang.org/protobuf/encoding/protojson"
```
Comment (reality95) [inline server/scheduler/scheduler.go:19]: Is this deprecated? I see this one is used in the services

**Review by reality95 — APPROVED** [review]

**Review by irmana — APPROVED** [review]
A well deserved upgrade of dependencies. Thumbs up!

File: server/scheduler/scheduler.go (hunk 1 above)
Comment (jkuri) [reply server/scheduler/scheduler.go]: no, actually `github.com/golang/protobuf` package is deprecated. I just updated everything to this package including `install_dependencies` script in `Makefile`. It should work now.

**Review by reality95 — DISMISSED** [review]

**Comment by jkuri** [comment]
Everything should be good now, I also tried to build both docker images and will push to registry after this PR will be merged. Please re-review.

**Review by irmana — APPROVED** [review]

**Pull request merged**